### PR TITLE
Update Extinction Curse Book 3 Ch2 NPCs

### PR DESCRIPTION
**Pull request by avagdumortesin**
Updated Caustic Dart Trap, Xulgath Bomber, and Xulgath Skirmisher.

### CODE REVIEW

File: packs/data/extinction-curse-bestiary.db/xulgath-bomber.json [hunk 1]
```diff
@@ -756,7 +754,7 @@
...
+                    "value": "<p>The @Compendium[pf2e.conditionitems.Sickened]{Sickened} condition from concentrated xulgath bile doesn't improve on its own until the target recovers from the poison. A target who recovers from concentrated xulgath bile is immune to being sickened by this poison for 1 minute;</p>\n<hr />\n<p><strong>Saving Throw</strong> @Check[type:fortitude|dc:22];</p>\n<p><strong>Maximum Duration</strong> 6 rounds;</p>\n<p><strong>Stage 1</strong> [[/r {1d10}[poison]]]{1d10 poison damage} and @Compendium[pf2e.conditionitems.Sickened]{Sickened 1} (1 round);</p>\n<p><strong>Stage 2</strong> [[/r {2d10}[poison]]]{2d10 poison damage} and @Compendium[pf2e.conditionitems.Sickened]{Sickened 2} (1 round);</p>\n<p><strong>Stage 2</strong> [[/r {2d10}[poison]]]{2d10 poison damage} and @Compendium[pf2e.conditionitems.Sickened]{Sickened 3} (1 round).</p>"
```
Comment (SpartanCPA) [inline packs/data/extinction-curse-bestiary.db/xulgath-bomber.json:757]:
```suggestion
                    "value": "<p>The @Compendium[pf2e.conditionitems.Sickened]{Sickened} condition from concentrated xulgath bile doesn't improve on its own until the target recovers from the poison. A target who recovers from concentrated xulgath bile is immune to being sickened by this poison for 1 minute</p>\n<p><strong>Saving Throw</strong> @Check[type:fortitude|dc:22]</p>\n<hr />\n<p><strong>Maximum Duration</strong> 6 rounds</p>\n<p><strong>Stage 1</strong> [[/r {1d10}[poison]]]{1d10 poison damage} and @Compendium[pf2e.conditionitems.Sickened]{Sickened 1} (1 round)</p>\n<p><strong>Stage 2</strong> [[/r {2d10}[poison]]]{2d10 poison damage} and @Compendium[pf2e.conditionitems.Sickened]{Sickened 2} (1 round)</p>\n<p><strong>Stage 2</strong> [[/r {2d10}[poison]]]{2d10 poison damage} and @Compendium[pf2e.conditionitems.Sickened]{Sickened 3} (1 round)</p>"
```
Modus operandi has been to place the hr element after the saving throw

We do not need the semicolons to end each line. Those are used to signify line breaks while saving print space, and we do not have that limitation

**Review by SpartanCPA — CHANGES_REQUESTED** [review]

File: packs/data/extinction-curse-bestiary.db/xulgath-bomber.json (hunk 1 above)
Comment (avagdumortesin) [reply packs/data/extinction-curse-bestiary.db/xulgath-bomber.json]: Fixed with commit https://github.com/foundryvtt/pf2e/pull/2727/commits/cc1c9753dcf9c6261e74295493b5f3a899de6b64

File: packs/data/extinction-curse-bestiary.db/xulgath-bomber.json [hunk 2]
```diff
@@ -1149,7 +1147,7 @@
                     "value": null
                 },
                 "description": {
-                    "value": "<p>A xulgath bomber's items listed as infused last for 24 hours, or until the next time they make their daily preparations.</p>"
+                    "value": "<p>A xulgath bomber carries the following infused items: 2 doses of concentrated xulgath bile, 2 moderate acid flasks, and 2 moderate alchemist's fires. These items last for 24 hours, or until the next time they make their daily preparations.</p>"
```
[the remark this answers is not in the document]
Comment (avagdumortesin) [reply packs/data/extinction-curse-bestiary.db/xulgath-bomber.json]: Fixed with commit https://github.com/foundryvtt/pf2e/pull/2727/commits/cc1c9753dcf9c6261e74295493b5f3a899de6b64

File: packs/data/extinction-curse-bestiary.db/xulgath-bomber.json [hunk 3]
```diff
@@ -546,7 +544,7 @@
                 }
             },
             "img": "systems/pf2e/icons/equipment/alchemical-items/alchemical-bombs/alchemists-fire.webp",
-            "name": "Alchemist's Fire (Moderate) [Infused]",
+            "name": "Alchemist's Fire (Moderate, Infused)",
```
[the remark this answers is not in the document]
Comment (avagdumortesin) [reply packs/data/extinction-curse-bestiary.db/xulgath-bomber.json]: Fixed with commit https://github.com/foundryvtt/pf2e/pull/2727/commits/cc1c9753dcf9c6261e74295493b5f3a899de6b64

File: packs/data/extinction-curse-bestiary.db/xulgath-bomber.json [hunk 4]
```diff
@@ -529,7 +528,6 @@
                         "bomb",
                         "consumable",
                         "fire",
-                        "infused",
                         "splash"
```
[the remark this answers is not in the document]
Comment (avagdumortesin) [reply packs/data/extinction-curse-bestiary.db/xulgath-bomber.json]: Fixed with commit https://github.com/foundryvtt/pf2e/pull/2727/commits/cc1c9753dcf9c6261e74295493b5f3a899de6b64

File: packs/data/extinction-curse-bestiary.db/xulgath-bomber.json [hunk 5]
```diff
@@ -377,7 +377,6 @@
                         "alchemical",
                         "bomb",
                         "consumable",
-                        "infused",
                         "splash"
```
[the remark this answers is not in the document]
Comment (avagdumortesin) [reply packs/data/extinction-curse-bestiary.db/xulgath-bomber.json]: Fixed with commit https://github.com/foundryvtt/pf2e/pull/2727/commits/cc1c9753dcf9c6261e74295493b5f3a899de6b64

File: packs/data/extinction-curse-bestiary.db/xulgath-bomber.json [hunk 6]
```diff
@@ -394,7 +393,7 @@
                 }
             },
             "img": "systems/pf2e/icons/equipment/alchemical-items/alchemical-bombs/acid-flask.webp",
-            "name": "Acid Flask (Moderate) [Infused]",
+            "name": "Acid Flask (Moderate, Infused)",
```
[the remark this answers is not in the document]
Comment (avagdumortesin) [reply packs/data/extinction-curse-bestiary.db/xulgath-bomber.json]: Fixed with commit https://github.com/foundryvtt/pf2e/pull/2727/commits/cc1c9753dcf9c6261e74295493b5f3a899de6b64

File: packs/data/extinction-curse-bestiary.db/caustic-dart-trap.json [hunk 7]
```diff
@@ -8,15 +8,15 @@
...
             "hp": {
-                "details": "",
+                "details": "per launching mechanism",
                 "max": 80,
                 "temp": 0,
                 "tempmax": 0,
                 "value": 80
             },
             "stealth": {
                 "details": "<p>(expert) or DC 30 (expert) to notice the three launching mechanisms built into the wall</p>",
```
[the remark this answers is not in the document]
Comment (avagdumortesin) [reply packs/data/extinction-curse-bestiary.db/caustic-dart-trap.json]: Fixed with commit https://github.com/foundryvtt/pf2e/pull/2727/commits/cc1c9753dcf9c6261e74295493b5f3a899de6b64

**Review by SpartanCPA — APPROVED** [review]

**Pull request merged**